src/app.js

```diff
@@ -27,3 +27,7 @@ ReactDOM.render(
   </Provider>,
   document.getElementById('app')
 );
+
+window.Intercom("boot", {
+  app_id: "scseqm3j"
```

Comment:
according to the Intercom docs, if the user is logged in, you can add more properties to this initialization step to better describe the user and provide a better experience.

how will you handle logged in users vs logged out users?